src/main/java/com/artipie/settings/AliasSettings.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.settings;
+
+import com.amihaiemil.eoyaml.Yaml;
+import com.artipie.asto.Key;
+import com.artipie.asto.Storage;
+import com.artipie.misc.ContentAsYaml;
+import hu.akarnokd.rxjava2.interop.SingleInterop;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Find aliases settings for repository.
+ * @since 0.28
+ */
+public final class AliasSettings {
```

Comment:
This class has only one method.
Looks like we can make this method static and move it to another place.

Reply:
I do not see what is wrong with the class (or interface) with one method. For example, `Slice`, `Response`, `Function` have one method, are there any problems?

This class has its individual purpose (to find aliases settings from storage) and corresponding test. Besides, public static methods are prohibited. But I think it would be better and more logical to make `Storage` field of the class instead of method parameter.

Reply:
> But I think it would be better and more logical to make Storage field of the class instead of method parameter.

ok